counter/package.json

```diff
@@ -1,8 +1,9 @@
 {
     "scripts": {
         "postinstall": "spago install",
-        "start": "spago build && concurrently \"spago build --watch\" \"parcel index.html\"",
-        "build": "spago build && parcel build index.html"
+        "bundle-ps": "spago bundle-module --to ./output/bundle.js",
```

Comment:
Would be curious to try this (again) on MyCV. Not sure if it cooperates with other stuff like JSX -> JS compilation though.

Reply:
Everything should work. The only difference is in how to address the entry point.